src/scala/io/bazel/rules_scala/scrooge_support/Compiler.scala

```diff
@@ -33,8 +33,9 @@ package io.bazel.rules_scala.scrooge_support
 
 import com.twitter.scrooge._
 import com.twitter.scrooge.ast.Document
-import com.twitter.scrooge.backend.{GeneratorFactory, ScalaGenerator, ServiceOption}
+import com.twitter.scrooge.backend.{GeneratorFactory, ScalaGenerator}
 import com.twitter.scrooge.frontend.{FileParseException, TypeResolver, ThriftParser, Importer, MultiImporter, ZipImporter}
```

Comment:
nit: as you're touching this file, please do some cleanup too: seems that Importer, MultiImporter, ZipImporter are unused imports